src/components/issues/Card.module.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+	padding: 1rem;
+	width: 100%;
+	background: #fff;
+	border: 1px solid #d9d9d9;
+	border-radius: 0.5rem;
+	transition: 0.2s ease-out;
+	display: flex;
+	flex-direction: column;
+	margin: 2rem auto;
+	justify-content: space-between;
+	position: relative;
+}
+
+.card:hover {
+	border-color: #e5e5e5;
+	filter: drop-shadow(0 20px 13px rgba(0, 0, 0, 0.02))
+		drop-shadow(0 8px 5px rgba(0, 0, 0, 0.06));
+}
+
+.card__top {
+	display: flex;
+	align-items: flex-start;
+	gap: 2rem;
+}
+
+.card__top__details {
+	flex: 1;
+	max-width: 75%;
+	width: 100%;
+	overflow-x: clip;
+}
+
+.card__top__details__meta_data {
+	margin: 0 0 0.75rem;
+}
+
+.card__top__details__meta_data h2 {
+	font-weight: 600;
+	font-size: 1.5rem;
+	margin: 0 0 0.25rem;
+	color: #1c1917;
+}
+
+.card__top__details__meta_data p {
+	font-size: 0.875rem;
+	font-weight: 500;
+	color: #78716c;
+}
+
```

Comment:
can nest the `p` and `h2` styles in `.card__top__details__meta_data` block

Reply:
Done